### DISCUSSION TITLE
Add + "create list" doesn't show yellow star

### DISCUSSION THREAD

**Discussion by chrisbuckleycode** [community/community#49747] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

If I add a repo to an existing list, the start will turn yellow.
But if I create the list for the first time to add a repo, the repo will get added but the star won't turn yellow. Have to refresh the page to see it yellow.
Great feature these lists are by the way!

**Comment by bbkx226** [comment]
Yes, that is the expected behavior in GitHub. When you add a repository to an existing list, the star next to the repository's name will turn yellow automatically, indicating that you have starred it.

However, when you create a new list and add a repository to it for the first time, the star next to the repository's name will not turn yellow automatically. You will need to refresh the page to see the change in the star color.

This is because the page needs to refresh to retrieve the updated information about the repository, including the fact that you have starred it. Once you refresh the page, the star should turn yellow, indicating that you have starred the repository.

**Comment by wellhungwguy** [comment]
- thats because lve been hacked several times since l first made the virus checker and web3 remover

**Reply by wellhungwguy** [reply]
```

- [ ] - - # > ```~~@chrisbuckleycode ~~

**Reply by wellhungwguy** [reply]
> - thats because lve been hacked several times since l first made the virus checker and web3 remover

**Comment by wellhungwguy** [comment]
> - thats because lve been hacked several times since l first made the virus checker and web3 remover

-

**Comment by wellhungwguy** [comment]
all crypto enthusists if you dont see  an actual value in your wallet it is because someone theif has gained access to your wallet and put a virus inside to either steal your coins or just to fuck with you so they can steal the real coins and replace them with fake ones this is what lve found in side my bit keep wallet and all so mu math wallet.,

**Reply by wellhungwguy** [reply]
when l find out whos doing it l cut your fucking tongue out and feed it to you then l cut one finger off at a time till l get all my crypto back or you will end up in tiny peices being feed to my animals...

**Reply by wellhungwguy** [reply]
Thats a promise .. lm a man of my word., l say it l ll fucken do it.

**Reply by Richmond574** [reply]
I'm on it

**Comment by AzizBenIsmail** [comment]
This seems to be a bug in the GitHub UI. It appears that the yellow star icon is not being updated immediately when you create a new list and add a repository to it for the first time. As you mentioned, refreshing the page should update the star icon and show it in yellow.

I suggest reporting this issue to GitHub support so that their team can investigate and fix the bug. To report an issue, go to the GitHub Support page and click on "Contact Us" at the bottom of the page. From there, you can select the category that best fits your issue and submit a report.

In the meantime, refreshing the page after creating a new list and adding a repository to it should update the star icon and show it in yellow, as you mentioned.
